package.json

```diff
@@ -40,12 +40,12 @@
     "build:dev": "npm run remove-rcue; REACT_APP_RCUE=false npm run build:kiali",
     "build:kiali": "npm run lint:prod && npm run copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts-ts build",
     "build:prod": "npm run restore-rcue; REACT_APP_RCUE=true npm run build:kiali",
-    "precommit": "(npm run prettier -- --list-different || (echo 'Above file(s) were modified by prettier, check them before commiting' && false) ) && npm run lint:prod",
+    "precommit": "yarn run pretty-quick -- --staged && npm run lint:prod",
```

Comment:
I think you need to add `pretty-quick` as a dev dependency.

Reply:
I still don't see `pretty-quick` in the dev-dependencies: https://github.com/azz/pretty-quick

Reply:
I get this error: 
`error Command "pretty-quick" not found.`